core/src/test/scala/unit/kafka/server/ClientQuotasRequestTest.scala

```diff
@@ -212,7 +214,9 @@ class ClientQuotasRequestTest extends BaseRequestTest {
           InetAddress.getByName(unknownHost)
         else
           InetAddress.getByName(entityName)
-        assertEquals(expectedMatches(entity), servers.head.socketServer.connectionQuotas.connectionRateForIp(entityIp), 0.01)
+        TestUtils.waitUntilTrue(
+          () => expectedMatches(entity) - servers.head.socketServer.connectionQuotas.connectionRateForIp(entityIp) < 0.01
+          ,"Broker didn't update quotas from Zookeeper")
```

Comment:
* We usually put the `,` at the end of the previous line when we break a line.
* Could we add the expected value and the current value in the message? We could say something like `Expected connection quota $expected for $entity, got $actual.`

Reply:
Thank you ! Both are very helpful . let me fix it .

Reply:
@dajac 
hello~
I remain the message `Broker didn't update quotas from Zookeeper` which clearly indicate the error.
And add  `expect` and `got` to error message

Can you help me take a look again

Reply:
Thanks for the update. I think that we should not mention ZK here as ZK is going away. How about `Connection quota of $entity is not $expected but $actual`?

Reply:
Thank you  ! I have fixed it .